### PR TITLE
Feature autoeject

### PR DESCRIPTION
**Pull request by vincent-miller**
Added AutoEject (autodumper)

### CODE REVIEW

File: QuasarHelios2015/src/org/team1540/quasarhelios/AutoEjector.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import ccre.channel.BooleanStatus;
+import ccre.ctrl.BooleanMixing;
+import ccre.igneous.Igneous;
+import ccre.instinct.AutonomousModeOverException;
+import ccre.instinct.InstinctModule;
+
+public class AutoEjector extends InstinctModule {
+	public BooleanStatus create() {
+		BooleanStatus b = new BooleanStatus(false);
+		AutoEjector a = new AutoEjector();
```
Comment (amackworth) [inline QuasarHelios2015/src/org/team1540/quasarhelios/AutoEjector.java]: Shouldn't this be ```QuasarHelios.autoLoader```?

Comment (amackworth) [inline QuasarHelios2015/src/org/team1540/quasarhelios/AutoEjector.java]: Sorry, scratch that!

**Comment by amackworth** [comment]
Looks good.

:sparkles: :+1: :sparkles:

File: QuasarHelios2015/src/org/team1540/quasarhelios/AutoEjector.java (hunk 1 above)
Comment (col6y) [inline QuasarHelios2015/src/org/team1540/quasarhelios/AutoEjector.java]: This is the autoEjector, not the autoLoader.

**Pull request merged**